README.md

```diff
@@ -1118,6 +1118,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
 
 ## Domain
   * [Freenom](https://freenom.com) — Free .tk,.ml,.ga,.cf and .gq domain.
+  * [Free Domains](https://github.com/free-domains/register) - Free subdomains for developers.
```

Comment:
Please note the exclusions when you opened the PR `* Free dns services that are generic frontends to CloudFlare or similar`

Reply:
@ripienaar It is a subdomain service where users can register domains such as `foo.is-a-frontend.dev`, it is not a DNS host.

Reply:
It is like is-a.dev and Open Domains.

Reply:
I know what it is, its a generic frontend to cloudflare dns, exactly as it says is not accepted.

Reply:
@ripienaar It's not a DNS host... I buy domains and let people create subdomains on them.

Reply:
Again, I know what it is, you dont need to explain.

Reply:
I don't understand why it's being rejected when it's the exact same as the other services.

Reply:
I also rejected several others of those already, leaving just the ones already on here.

It adds zero value, the services are not unique, you just copy each others code and put it there and come here presenting it as a SaaS when yo have like 1 user.  Generally the people who run them think its ok to come here and argue and be general nuisances, so I stopped accepting them.

Please accept the rejection wthout further arguiing

Reply:
Ok